controller/workitem_blackbox_test.go

```diff
@@ -559,6 +559,37 @@ func (s *WorkItemSuite) TestUpdateWorkitemWithoutReorder() {
 	assert.Equal(s.T(), wi.Data.Attributes[workitem.SystemOrder], updated.Data.Attributes[workitem.SystemOrder])
 }
 
+// TestReorderDoesNotUpdateAnyOtherWIFields tests that no other workitem fields are updated while reorder
+func (s *WorkItemSuite) TestReorderDoesNotUpdateOtherWIFields() {
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment())
+
+	// create workitems
+	payload := minimumRequiredCreateWithTypeAndSpace(fxt.WorkItemTypes[0].ID, fxt.Spaces[0].ID)
+	originalTitle := "Original Title"
+	payload.Data.Attributes[workitem.SystemTitle] = originalTitle
+	payload.Data.Attributes[workitem.SystemState] = workitem.SystemStateClosed
+	_, workitem1 := test.CreateWorkitemsCreated(s.T(), s.svc.Context, s.svc, s.workitemsCtrl, *payload.Data.Relationships.Space.Data.ID, &payload)
+	_, workitem2 := test.CreateWorkitemsCreated(s.T(), s.svc.Context, s.svc, s.workitemsCtrl, *payload.Data.Relationships.Space.Data.ID, &payload)
+
+	titleNotUpdated := "This Title Should Not Be Updated"
+	workitem1.Data.Attributes[workitem.SystemTitle] = titleNotUpdated
+
+	payload2 := minimumRequiredReorderPayload()
+
+	var dataArray []*app.WorkItem // dataArray contains the workitem(s) that have to be reordered
+	dataArray = append(dataArray, workitem1.Data)
+	payload2.Data = dataArray
+	payload2.Position.ID = workitem2.Data.ID // Position.ID specifies the workitem ID above or below which the workitem(s) should be placed
+	payload2.Position.Direction = string(workitem.DirectionAbove)
```

Comment:
`workitem.DirectionAbove` is already a string, you don't need the `string(...)`

Reply:
`workitem.DirectionAbove` is `workitem.DirectionType`

Reply:
Yes, but `workitem.DirectionType` is of type `string`
See https://play.golang.org/p/Nov7b0810R-
```
package main

import (
	"fmt"
)

func main() {
	type direction string
	const foo direction = "above"
	bar := foo
	fmt.Println(foo, bar)
}
```